fe/fe-core/src/main/java/org/apache/doris/load/StreamLoadRecordMgr.java

```diff
@@ -0,0 +1,112 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.load;
+
+import com.google.common.collect.ImmutableMap;
+import org.apache.doris.catalog.Catalog;
+import org.apache.doris.common.ClientPool;
+import org.apache.doris.common.Config;
+import org.apache.doris.plugin.AuditEvent;
+import org.apache.doris.plugin.AuditEvent.EventType;
+import org.apache.doris.plugin.StreamLoadAuditEvent;
+import org.apache.doris.system.Backend;
+import org.apache.doris.thrift.BackendService;
+import org.apache.doris.thrift.TNetworkAddress;
+import org.apache.doris.thrift.TStreamLoadRecord;
+import org.apache.doris.thrift.TStreamLoadRecordResult;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.ReentrantReadWriteLock;
+
+public class StreamLoadRecordMgr {
+    private static final Logger LOG = LogManager.getLogger(StreamLoadRecordMgr.class);
+    private ReentrantReadWriteLock lock = new ReentrantReadWriteLock();
+
+    public StreamLoadRecordMgr() {
+        Thread pullStreamLoadRecordThread = new Thread(new PullStreamLoadRecordThread());
+        pullStreamLoadRecordThread.start();
```

Comment:
Better not start the worker thread in constructor. Use a `start()` method instead.
And I think this thread can only be started in Master FE?

Reply:
> Better not start the worker thread in constructor. Use a `start()` method instead.
> And I think this thread can only be started in Master FE?

Thanks. I have modified the code.